cmd/manager/operator.go

```diff
@@ -313,10 +327,25 @@ func RunOperator(cmd *cobra.Command, args []string) {
 	}
 	// We need to set PLATFORM env var if the PLATFORM flag is set
 	pflag := os.Getenv("PLATFORM")
+	if pflag == "" {
+		clusterClaim := &clusterv1alpha1.ClusterClaim{}
+		if err := kubeClient.RESTClient().Get().RequestURI("/apis/cluster.open-cluster-management.io/v1alpha1/clusterclaims/product.open-cluster-management.io").Do(ctx).Into(clusterClaim); err != nil {
+			setupLog.Info("Couldn't get ClusterClaim. This is not fatal though.")
```

Comment:
Since we're using `ClusterClaim` as an internal implementation detail, we could say something like:

```go
setupLog.Info("Unable to determine managed offering type.")
```

Then if readers see the error, but they're using self-managed, they don't really have a reason to be alarmed.

Reply:
agree, let me fix it